about/help.md

```diff
@@ -0,0 +1,33 @@
+## New to Julia?
```

Comment:
Given that you have a bunch of lists here, and that using `@@tight-list` is annoying, you could just put this at the top:

```
~~~
<style>
  .container li > p {margin-bottom: 0;}
</style>
~~~
```

The reason for this is that the Julia Markdown to HTML processor adds `<p></p>` in  list items, these inherit properties for the whole document and it leads to unsightly big vertical spaces between list items.